.env.sepolia

```diff
@@ -2,6 +2,7 @@ GENESIS_FILE_PATH=sepolia/genesis.json
 SEQUENCER_HTTP=https://rpc.sepolia-api.lisk.com
 OP_NODE_OVERRIDE_CANYON=0
 RETH_CHAIN=lisk-sepolia
+OVERRIDE_FJORD=1716998400
```

Comment:
where did this number come from?

Reply:
https://docs.optimism.io/builders/node-operators/network-upgrades#activations